website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4061,6 +4203,18 @@ purchases.apply(Group.byFieldNames("userId")
     .aggregateField("costCents", Top.<Long>largestLongsFn(10), "topPurchases"));
 {{< /highlight >}}
 
+{{< highlight python >}}
+input_pc = ... # {"user_id": ...,"item_Id": ..., "cost_cents": ...}
+output_pc = input_pc | beam.GroupBy("user_id")
+	.aggregate_field("item_id",count,"num_purchases")
+	.aggregate_field("cost_cents",sum,"total_spendcents")
```

Comment:
nit: `sum` will work here since it's a built-in that we detect, but you need to define `count`. What you want here is the CombineFn used in the [`Count` transform](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.combiners.html#apache_beam.transforms.combiners.Count).

Unfortunately it looks like we don't expose the CombineFn like Java does via `Count.combineFn`. Unless I'm missing something.. do you know @robertwb or @tvalentyn?

Reply:
So I can't use count directly? (other than defining the count function)
are there any other aggregate functions that can be used which need not be defined? I guess we can use max/min here

Reply:
there is https://github.com/apache/beam/blob/master/sdks/python/apache_beam/transforms/combiners.py#L158 , but I am not sure what exactly is needed to 'expose' this CombineFn - perhaps you could file a FR @TheNeuralBit ?

Reply:
Good point, I filed https://github.com/apache/beam/issues/23511